src/main/kotlin/uk/gov/justice/digital/hmpps/incidentreporting/service/EventPublishAndAuditService.kt

```diff
@@ -19,26 +19,25 @@ class EventPublishAndAuditService(
     auditData: Any? = null,
     source: InformationSource,
   ) {
-    publishEvent(event = eventType, reportId = reportId, source = source)
+    domainEvent(eventType = eventType, reportId = reportId, source = source)
 
     auditData?.let {
       auditEvent(
         auditType = eventType.auditType,
         id = reportId.toString(),
         auditData = it,
-        source = source,
       )
     }
   }
 
-  private fun publishEvent(
-    event: ReportDomainEventType,
+  private fun domainEvent(
```

Comment:
I think we should retain the "publish" in this function name, maybe rename to `publishDomainEvent()`, but I like the "publish" in it as it makes it clear what this is doing, especially in the client code, e.g.

```
domainEvent(eventType = eventType, reportId = reportId, source = source)
```

seeing this doesn't tell me much, it almost seems like it's constructing an object but not using it.